### DISCUSSION TITLE
Upgrading to v6

### DISCUSSION THREAD

**Discussion by danielbattat** [react-hook-form/react-hook-form#3319] [Q&A] [answered]
When I upgrade to v6 with npm, and run my app, I receive a "Dependency was not found" error for "react-hook-form". Even using a simple "getting started" code example produces this error.

I tried deleting node_modules and that did not help. Any insight appreciated.

**Comment by bluebill1049** [comment]
not sure why on this.

**Reply by bluebill1049** [reply]
can you reproduce that in CSB?

**Reply by danielbattat** [reply]
Thanks for the response. Haven't been able to reproduce on CSB :(

**Comment by danielbattat** [comment, marked as answer]
This resolved my issue: https://github.com/react-hook-form/react-hook-form/issues/56#issuecomment-653720510
